content/2017/08-not-all-those-who-wander-are-lost.rst

```diff
@@ -102,6 +83,11 @@ Tooling
 Events / Meetups
 ==================
 
+- `NixCon 2017`_ October 28-31, Munich.
```

Comment:
The nixcon should probably be featured a lot more prominent, since it’s going to be the biggest community event this year.